webpack.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  entry: __dirname + '/client/src/index.jsx',
+  module: {
+    rules: [
+      {
+        test: [/\.jsx$/],
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-react', '@babel/preset-env'],
```

Comment:
@janicelam21 

Out of curiosity--by having your presets here vs. a .babelrc file, did you also have to articulate your presets in your jest configuration file?

Reply:
I still haven't gotten Jest to work for reach, so maybe this was the issue? I'll put it in its own config file and see if it fixes the errors for Jest.